esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
I don't see any benefits of having Chrome 52-53 as well. More than that someone may need to mention `chrome53` in table in any time later. Also it will be stable soon.